gpii/node_modules/flowManager/src/RepositorySolutionsLoader.js

```diff
@@ -0,0 +1,115 @@
+/*!
+GPII Full SHA Revision DataSource
+
+Copyright 2020 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    path = require("path"),
+    gpii = fluid.registerNamespace("gpii");
+
+require("kettle");
+
+fluid.registerNamespace("gpii.flowManager.repositorySolutionsLoader");
+
+fluid.defaults("gpii.flowManager.repositorySolutionsLoader", {
+    gradeNames: ["kettle.dataSource.URL"],
+    urlTemplate: "%prefix/%revision/%suffix/%fileName",
+    url: {
+        expander: {
+            funcName: "fluid.stringTemplate",
+            args: ["{that}.options.urlTemplate", {
+                prefix: "{that}.options.urlPrefix",
+                suffix: "{that}.options.urlSuffix"
+            }]
+        }
+    },
+    termMap: {
+        revision: "%directRevision",
+        fileName: "%directFileName"
+    },
+    // The URL prefix and suffix are distributed from a config file such as
+    // e.g., gpii.flowManager.config.local.base, as is the path to the local
+    // folder in which to save the contents of the downloaded solutions registry
+    // file.
+    urlPrefix: "",
+    urlSuffix: "",
+    solutionsRegistryFolderPath: "",
```

Comment:
This folder path doesn't see to be use anywhere. According to your comment, this path is intended to be used to save the contents of the downloaded solutions registry to the local system. I doubt if we should ever save the downloaded SR to the local disk, especially it's pointing to `"%gpii-universal/testData/solutions"` in [the config file](https://github.com/GPII/universal/pull/840/files#diff-ce29f5ee38b24db477e76b84aeadfe6dR21-R23), which means to overwrite the SR in the universal code base.

Reply:
`solutionsRegistryFolderPath` is not used in the functional code. Shall it be removed? If it can be removed, please also clean up its appearances in tests and config files. 

Please also revisit the code comment above about "save the contents of the downloaded solutions registry file". Thanks.

Reply:
Okay removed, but it was tricky finding all of them.